src/main/java/de/qabel/desktop/ui/tray/TrayIconMouseAdapter.java

```diff
@@ -0,0 +1,103 @@
+package de.qabel.desktop.ui.tray;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.MouseAdapter;
+import java.awt.event.MouseEvent;
+import java.util.ArrayList;
+
+class TrayIconMouseAdapter extends MouseAdapter {
+    private final JPopupMenu popup;
+    private final QabelTray qabelTray;
+    private boolean visible;
+
+    TrayIconMouseAdapter(QabelTrayImpl qabelTray) {
+        this.qabelTray = qabelTray;
+        this.popup = qabelTray.popup;
+    }
+
+    @Override
+    public void mouseReleased(MouseEvent e) {
+        if (isDoubleLeftClick(e)) {
+            qabelTray.bringAppToFront();
+        } else if (isRightClick(e)) {
+            Point position = calculatePosition(e);
```

Comment:
This doesn't check if it is a left click. A double right click would return true, too.